images/capi/ansible/roles/providers/tasks/vmware.yml

```
@@ -36,6 +36,18 @@
     - cloud-utils-growpart
   when: ansible_os_family == "RedHat"
 
+- name: Install cloud-init packages
+  command: tdnf install {{ packages }} -y
+  vars:
+    packages: "cloud-init cloud-utils python3-netifaces"
+  when: ansible_os_family == "VMware Photon OS"
+
+- name: copy cloud-init 19.1-2 rpm
+  command: curl -LO https://vmware.bintray.com/photon_dev_x86_64/noarch/cloud-init-19.1-2.ph3.noarch.rpm
+
+- name: Update cloud-init 18.4
```

This comment says 18.4
```suggestion
- name: Update cloud-init to 19.1-2
```

---

done